resources/assets/js/annotations/annotatorContainer.vue

```diff
@@ -160,6 +160,9 @@ export default {
         annotationsHiddenByFilter() {
             return this.annotations.length !== this.filteredAnnotations.length;
         },
+        getAnnotationCount() {
```

Comment:
Since this is a computed property, I would not give it a name that sounds like a method. You could call it `annotationCount`.